pom.xml

```diff
@@ -1860,43 +1874,6 @@
       </activation>
     </profile>
 
-    <!-- spark3 is an alias of spark3.3 -->
-    <profile>
-      <id>spark3</id>
-      <properties>
-        <spark3.version>3.3.0</spark3.version>
-        <spark.version>${spark3.version}</spark.version>
-        <sparkbundle.version>3</sparkbundle.version>
```

Comment:
i think this profile still needed to generate hudi-spark3-bundle as a legacy name

Reply:
We can just copy 3.3 one, right?